### PR TITLE
Update readme about deployment

### PR DESCRIPTION
**Pull request by patch0**
Add a bit of clue based on recent experience on how to get the thing deployed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,3 +176,17 @@ command.
...
 
 Check out our [troubleshooting](troubleshooting.md) doc.
+
+## Deployment
+
+In order to get your changes deployed there might be several merges/upgrades to sort out.  For example, if you update a banner, requiring changes in `cp-translations`, `cp-zen-frontend`, and `cp-zen-platform`, you need to:
+
+1. Merge `cp-translations` changes.  Re-pull your master branch to get the auto-incremented version.
+2. Update the version of `cp-translations` in packages.json for `cp-zen-frontend`, and re-install yarn dependencies, and get them merged to master.  Re-puu your master branch to get the auto-incremented version number.
+3. Finally update the version of `cp-zen-frontend` in `cp-zen-platform`, re-install yarn dependencies, and then get those changes merged to master.
```
Comment (glenpike) [inline README.md:186]: Need to update the version of cp-translations in cp-zen-platform maybe too?

File: README.md [hunk 2]
```diff
@@ -176,3 +176,17 @@ command.
 ### Still having issues?
 
 Check out our [troubleshooting](troubleshooting.md) doc.
+
+## Deployment
+
+In order to get your changes deployed there might be several merges/upgrades to sort out.  For example, if you update a banner, requiring changes in `cp-translations`, `cp-zen-frontend`, and `cp-zen-platform`, you need to:
+
+1. Merge `cp-translations` changes.  Re-pull your master branch to get the auto-incremented version.
+2. Update the version of `cp-translations` in packages.json for `cp-zen-frontend`, and re-install yarn dependencies, and get them merged to master.  Re-puu your master branch to get the auto-incremented version number.
```
Comment (glenpike) [inline README.md:185]: Not sure what 'Re-puu' is?

Comment (patch0) [reply README.md:185]: How do you even read?

File: README.md (hunk 1 above)
Comment (patch0) [reply README.md:186]: Does that not get pulled in automatically?..?

File: README.md [hunk 3]
```diff
@@ -176,3 +176,17 @@ command.
 ### Still having issues?
 
 Check out our [troubleshooting](troubleshooting.md) doc.
+
+## Deployment
+
+In order to get your changes deployed there might be several merges/upgrades to sort out.  For example, if you update a banner, requiring changes in `cp-translations`, `cp-zen-frontend`, and `cp-zen-platform`, you need to:
+
+1. Merge `cp-translations` changes.  Re-pull your master branch to get the auto-incremented version.
```
Comment (glenpike) [inline README.md:184]:
```suggestion
1. Merge `cp-translations` changes, wait for the CI to rebuild and publish a new npm package. Re-pull your master branch to get the auto-incremented version.
```

File: README.md [hunk 4]
```diff
@@ -176,3 +176,17 @@ command.
...
+2. Update the version of `cp-translations` in packages.json for `cp-zen-frontend`, and re-install yarn dependencies, and get them merged to master.  Re-pull your master branch to get the auto-incremented version number.
```
Comment (glenpike) [inline README.md:185]:
```suggestion
2. Update the version of `cp-translations` in packages.json for `cp-zen-frontend`, and re-install yarn dependencies, and get them merged to master. Wait for the CI to rebuild and publish a new npm package.   Re-pull your master branch to get the auto-incremented version number.
```

**Review by glenpike — COMMENTED** [review]
LGTM

**Pull request merged**